core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -865,6 +868,14 @@
           + " To add a summarizer set "
           + "`table.summarizer.<unique id>=<summarizer class name>.` If the summarizer has options"
           + ", then for each option set" + " `table.summarizer.<unique id>.opt.<key>=<value>`."),
+  @Experimental
+  TABLE_DELETE_BEHAVIOR("table.delete.behavior", DeletingIterator.Behavior.PROCESS.name(),
+      PropertyType.STRING,
+      "This determines what action to take when a delete marker is seen."
+          + " Valid actions are PROCESS and FAIL with PROCESS being the default.  When set to "
+          + "PROCESS, deletes will supress data.  When set to FAIL, any deletes seen will cause an"
+          + " exception. The purpose of FAIL is to support tables that never delete data and need "
+          + "fast seeks within the timestamp range of a column."),
```

Comment:
For consistency, should make the values lowercase like TABLE_FILE_COMPRESSION_TYPE... "Possible values: zstd, gz, snappy, lzo, none".  I don't think we have any uppercase property values...

Reply:
Could make it case-insensitive.

Reply:
It is case insensitive.  It calls toUpperCase when parsing it.